lib/role.rb

```diff
@@ -4,11 +4,17 @@
 # convenience methods for testing the role status of users.
 module Role
 
+  def grant_role!(role, facility: nil)
```

Comment:
One other problem with that method is it allows you to create duplicate role assignments. E.g. a user could end up with the Facility Director role twice.